axi_intercon_gen.py

```diff
@@ -0,0 +1,552 @@
+#!/usr/bin/env python
```

Comment:
Can we move the two Python scripts to the `scripts` directory? That's where we keep tool-specific scripts.

Reply:
Sure thing